### DISCUSSION TITLE
[Question]: How to enable AWS Bedrock

### DISCUSSION THREAD

**Discussion by MaxwellEdisons** [danny-avila/LibreChat#5121] [Q&A]
### What is your question?

I configured ENDPOINTS=openAI,google,gptPlugins,AWS Bedrock in the .env file, but I cannot select AWS Bedrock. How can I enable AWS Bedrock?

### More Details

AWS Bedrock

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by khfung** [comment]
you need to add 
BEDROCK_AWS_DEFAULT_REGION=us-east-1
BEDROCK_AWS_ACCESS_KEY_ID=your_access_key_id
BEDROCK_AWS_SECRET_ACCESS_KEY=your_secret_access_key
BEDROCK_AWS_MODELS=
on your .env file also

**Reply by MaxwellEdisons** [reply]
> you need to add BEDROCK_AWS_DEFAULT_REGION=us-east-1 BEDROCK_AWS_ACCESS_KEY_ID=your_access_key_id BEDROCK_AWS_SECRET_ACCESS_KEY=your_secret_access_key BEDROCK_AWS_MODELS= on your .env file also

I have set it up, but no available models are shown in the page. Is there somewhere else I need to set up?

**Comment by danny-avila** [comment]
https://www.librechat.ai/docs/configuration/pre_configured_ai/bedrock

**Comment by jr4t-ksb** [comment]
AWS Bedrock is incorrect.
It works with bedrock.
`ENDPOINTS=google,azureOpenAI,bedrock`

**Comment by AndresDevvv** [comment]
I also dont see it after adding it to the env even after restarting
